content/docs/command-reference/queue/index.md

```diff
@@ -0,0 +1,39 @@
+# queue
+
+A set of commands to manage the
+[DVC experiments](/doc/user-guide/experiment-management/experiments-overview)
+task queue: [start](/doc/command-reference/queue/start),
+[stop](/doc/command-reference/queue/stop),
+[status](/doc/command-reference/queue/status),
+[logs](/doc/command-reference/queue/logs),
+[remove](/doc/command-reference/queue/remove),
+[kill](/doc/command-reference/queue/kill)
+
+## Synopsis
+
+```usage
+usage: dvc queue [-h] [-q | -v]
```

Comment:
Need to add all queue commands in https://github.com/iterative/gatsby-theme-iterative/blob/main/packages/gatsby-theme-iterative/config/prismjs/dvc-commands.js for proper highlighting

Reply:
Would be great to get this in while this PR is waiting @pmrowla 🙏🏼

Reply:
https://github.com/iterative/gatsby-theme-iterative/pull/56